exec/java-exec/src/main/java/org/apache/drill/exec/ops/FragmentStats.java

```diff
@@ -31,6 +32,13 @@
 public class FragmentStats {
 //  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(FragmentStats.class);
 
+  //Skip operators that already have stats reported by org.apache.drill.exec.physical.impl.BaseRootExec
+  private static final List<Integer> operatorStatsInitToSkip = Lists.newArrayList(
```

Comment:
This could get out of sync with the types of senders that extend BaseRootExec.

Reply:
The add-on commit refactors by having the BaseRootExec constructor handle the substition without risking going out of sync with other senders extending BaseRootExec.